tools/api-markdown-documenter/api-report/api-markdown-documenter.api.md

```diff
@@ -151,12 +151,27 @@ export namespace DefaultDocumentationSuiteOptions {
 }
 
 // @public
-export interface DocumentationLiteralNode<T = unknown> extends Literal<T>, DocumentationNode {
+export interface DocumentationLiteralNode<TValue = unknown> extends Literal<TValue>, DocumentationNode {
+    readonly isLiteral: true;
+    readonly isParent: false;
     readonly type: string;
+    readonly value: TValue;
```

Comment:
Note: this property already exists on `Literal` - this change makes the property readonly, and surfaces it here directly for discoverability and documentation purposes.